HHS/Head-Start-TTADP
# [PROD] - Enable Dev & Staging email; Use transactions for all queries; Prevent intermittent server crashes on delete

## Description of change

TTAHUB-783: Email notifications were working in Sandbox, but we were still getting `ssl3_get_record:wrong version number` error in Staging and Dev. 783 addresses this.

TTAHUB-0: Prevent intermittent server crashes on activity reports deletions.

TTAHUB-657: Use transactions automatically for all queries.


## How to test
TTAHUB-783: In Dev, create an activity report and assign yourself as a collaborator. Verify that an email is received.
https://tta-smarthub-dev.app.cloud.gov/

TTAHUB-657: Regression testing only.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-783
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-657

## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
